app/src/components/widgets/charts/issuesStatusPageChart/constants.js

```diff
@@ -3,28 +3,34 @@ import { getItemColor } from 'components/widgets/charts/common/utils';
 
 export const AUTOMATION_BUG = 'automationBug';
 export const TO_INVESTIGATE = 'toInvestigate';
-export const DEFECT_TYPES = [AUTOMATION_BUG, TO_INVESTIGATE];
+export const PRODUCT_BUG = 'productBug';
+export const DEFECT_TYPES = [AUTOMATION_BUG, TO_INVESTIGATE, PRODUCT_BUG];
 
 export const COLORS = {
   [AUTOMATION_BUG]: getItemColor({ defectType: AUTOMATION_BUG }),
   [TO_INVESTIGATE]: getItemColor({ defectType: TO_INVESTIGATE }),
+  [PRODUCT_BUG]: getItemColor({ defectType: PRODUCT_BUG }),
```

Comment:
Please specify this color in the `common/constants/colors.js` (you can reuse `COLOR_PRODUCT_BUG`)

Reply:
Already fixed in another PR (EPMRPP-37000)